fastlane/docs/Actions.md

```diff
@@ -971,14 +971,17 @@ This action allows you to upload symbolication files to Sentry.
 
 ```ruby
 upload_symbols_to_sentry(
-  api_key: '...',
+  api_key: '...', # Do not use if using auth_token
```

Comment:
We should just list the one option that is recommended by sentry instead, so that the user doesn't have to choose